examples/01-module-declarations-reified-as-module-objects.md

```diff
@@ -2,17 +2,17 @@
 
 $$ {\text{module declarations} \over \text{module expressions}} = {\text{function declarations} \over \text{function expressions}} $$
 
-Except for the "they are static declaration parts", module declarations behave exactly like module expressions assigned to a `const` variable.
+Except for the "they are static declaration parts", module declarations behave exactly like module expressions assigned to a `const` variable, except that they are hoisted similarly to function declarations.
```

Comment:
We should probably say, "strict mode function declarations", so that this isn't taken to include Annex B 3.3 :)